cmd/lncli/routerrpc_active.go

```diff
@@ -6,5 +6,6 @@ import "github.com/urfave/cli"
 
 // routerCommands will return nil for non-routerrpc builds.
 func routerCommands() []cli.Command {
-	return []cli.Command{queryMissionControlCommand, resetMissionControlCommand}
+	return []cli.Command{queryMissionControlCommand,
```

Comment:
Nit: can place each one of these on its own line to help readability.

Reply:
yes, nicer. fixed